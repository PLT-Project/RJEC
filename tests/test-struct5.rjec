struct foo {
    x bool;
    y int;
    z bool;
}

func main() {
    var i struct foo;
    i = struct foo {
        x : false,
        z : true
    };
    printb(i.x);
    printi(i.y);
    printb(i.z);

    i.y = 10;
    
    printb(i.x);
    printi(i.y);
    printb(i.z);
}