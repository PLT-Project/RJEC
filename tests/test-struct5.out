false
0
true
false
10
true